synthpop/census_helpers.py

```diff
@@ -49,23 +49,23 @@ def _scale_and_merge(self, df1, tot1, df2, tot2, columns_to_scale,
         return df
 
     def block_group_query(self, census_columns, state, county, tract=None,
-                          year=None, id=None):
+                          year=2016, id=None):
         if id is None:
             id = "*"
         return self._query(census_columns, state, county,
                            forstr="block group:%s" % id,
-                           tract=tract, year=year)
+                           tract=tract, year=2016)
```

Comment:
`year=2016` in this line can be `year=year`, since year is one of the function input parameters (line 52) and has a default arg value of 2016